### ISSUE TITLE
no-standalone-expect: false positive error

### ISSUE THREAD

**Issue by akellbl4** [vitest-dev/eslint-plugin-vitest#789]
Example
```ts
import { it } from "vitest"

function fn(value: string) {
  return value === "a" || value === "b"
}

describe("tests", () => {
  it.for(["a", "b"])("test", (value, { expect }) => {
    expect(fn(value)).toBe(true)
  })
})

```
